site/specs-source/voice.json

```diff
@@ -2667,7 +2667,7 @@
             "minimum": 1,
             "maximum": 5,
             "default": 5,
-            "description": "The priority of this call over other calls from your account when outbound call queueing is enabled. For example, if during a call your application needs to place a new call and bridge it with the current call, you might want to create the call with priority 1 so that it will be the next call picked off your queue, after any previously queued priority 1 calls and ahead of other less time sensitive calls.  A lower value means higher priority, so a priority 1 call takes precedence over a priority 2 call. Range: integer values between 1 - 5. Default value is 5."
+            "description": "The priority of this call over other calls from your account when <a href='/docs/voice/rateLimits'>outbound call queueing</a> is enabled. For example, if during a call your application needs to place a new call and bridge it with the current call, you might want to create the call with priority 1 so that it will be the next call picked off your queue, after any previously queued priority 1 calls and ahead of other less time sensitive calls.  A lower value means higher priority, so a priority 1 call takes precedence over a priority 2 call. Range: integer values between 1 - 5. Default value is 5."
```

Comment:
This isn't what I meant, this is still hidden down in a random field description. Call queueing is a HUGE change to the way we send outbound calls. Information about call queueing needs to be first class on this page, not hidden in a link in the description of a field. This is a fine change, more links are good. I just really think there needs to be front and center text on this page describing the call queueing behavior in detail because this is the endpoint that that's relevant to.